kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/reflect/KotlinJsonAdapterTest.kt

```diff
@@ -950,4 +950,16 @@ class KotlinJsonAdapterTest {
     assertThat(adapter.fromJson("null")).isNull()
     assertThat(adapter.toJson(null)).isEqualTo("null")
   }
+
+  @Test fun kotlinClassesWithoutAdapterAreRefused() {
+    val moshi = Moshi.Builder().build()
+    try {
+      moshi.adapter<PlainKotlinClass>(PlainKotlinClass::class.java)
+      throw AssertionError("Should not pass here")
```

Comment:
nit: use `fail()`? for consistency with all the other test cases.

Reply:
derp, good call 7058846